### PR TITLE
added meteor client-side script for LoginState.get()

### PR DESCRIPTION
**Pull request by thinkholic**
also, updated README.md.

### CODE REVIEW

File: includes/login_state.js [hunk 1]
```diff
@@ -1,27 +0,0 @@
-LoginState = {};
```
Comment (arunoda) [inline includes/login_state.js]: Why removed this? Where' the new file?

Comment (thinkholic) [inline includes/login_state.js]: new file is `client/login_state.js. also changed on bower and node package.json files

**Comment by arunoda** [comment]
Is it possible to have a test case?

**Comment by thinkholic** [comment]
- Wrote a test case for `LoginState.get()`
- Fixed URLEncode Issue

**Pull request merged**